pkg/kubectl/cmd/top/top_pod_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cmd
+package top
```

Comment:
I agree with what @alexandercampbell says.
It will be good if someone can fix the naming for `set`.